homeassistant/components/poolsense/config_flow.py

```diff
@@ -19,12 +19,8 @@ class PoolSenseConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     VERSION = 1
     CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
 
-    _options = None
-
     def __init__(self):
         """Initialize PoolSense config flow."""
-        self._email = None
-        self._password = None
         self._errors = {}
```

Comment:
`_errors` can be a local variable now.

Reply:
Thanks. Do I need to create a new PR to correct this or can I push it to this branch?

Reply:
This PR is merged so a new PR is required. Thanks!